src/Sulu/Component/Content/Tests/Functional/Rlp/Mapper/PhpcrMapperTest.php

```diff
@@ -240,25 +295,18 @@ public function testMoveTwice()
         // FIXME after change mixin works: $this->assertEquals('sulu:history', $oldNodeMixins[0]->getName());
 
         $this->assertEquals($newNode, $oldNode->getPropertyValue('sulu:content'));
-        $this->assertEquals($this->content1, $newNode->getPropertyValue('sulu:content'));
+        $this->assertEquals(
+            $this->documentInspector->getNode($this->document1),
+            $newNode->getPropertyValue('sulu:content')
+        );
 
         // get content from new path
         $result = $this->rlpMapper->loadByResourceLocator('/products/asdf/content2-news', 'sulu_io', 'de');
-        $this->assertEquals($this->content1->getIdentifier(), $result);
+        $this->assertEquals($this->document1->getUuid(), $result);
 
         // get content from history should throw an exception
         $this->setExpectedException('Sulu\Component\Content\Exception\ResourceLocatorMovedException');
-        $result = $this->rlpMapper->loadByResourceLocator('/products/news/content1-news', 'sulu_io', 'de');
-    }
-
-    public function testMoveNotExist()
```

Comment:
This test doesn't make any sense anymore, since the `move` method does not exist anymore in favour of the `save` method.